source/faq.rst

```diff
@@ -216,5 +216,44 @@ You can change the minimum resolution for metrics by the following way:
    when |pmm-server| and |pmm-client| are on different networks,
    or when :ref:`pmm.amazon-rds`.
 
+.. _alertmanager:
+
+How to set up Alerting in PMM?
+================================================================================
+
+You can make PMM Server firing alerts when your monitored service reaches some thresholds in two ways:
+
+* using `Grafana Alerting feature <https://grafana.com/docs/grafana/latest/alerting/rules/>`_,
+* using external `Alertmanager <https://github.com/prometheus/alertmanager>`_ (a
+  high-performance solution developed by the Prometheus project to handle alerts sent
+  by Prometheus).
+
+Both options can be considered advanced features and require knowledge of
+third-party documentation.
+
+Either with Grafana Alerting or with Alertmanager you need to configure some
+alerting rule to define conditions under which the alert should be fired out,
+and some channel to which the alert is to be sent (e.g. e-mail).
+
+Grafana Alerts can be preferred because of being integrated into the PMM Server,
+while Alertmanager allows creating more sophisticated alerting rules and is
+much more manageable in setups with a large number of hosts.
```

Comment:
'while Alertmanager allows the creation of more sophisticated alerting rules and can be easier to manage installations with a large number of hosts; this additional flexibility comes at the expense of simplicity and requires advanced knowledge of Alertmanager rules.  Currently Percona cannot offer support for creating custom rules so you should already have a working Alertmanager instance prior to using this feature, however we are working hard to bring an integrated Alertmanager solution to make rule generation easy!'